app/views/categories/show.html.erb

```diff
@@ -1,19 +1,19 @@
 <div class="menu">
-<a href="/categories" class="text-light"><svg xmlns="http://www.w3.org/2000/svg" width="26" height="36" fill="currentColor" class="bi bi-arrow-left" viewBox="0 0 16 16">
-  <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
-</svg></a>
+  <a href="/categories" class="text-light"><svg xmlns="http://www.w3.org/2000/svg" width="26" height="36" fill="currentColor" class="bi bi-arrow-left" viewBox="0 0 16 16">
+      <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
+    </svg>
+  </a>
 <h2 class="text-light text-center">Details</h2>
 <span></span> 
 </div>
 <div class="col-10 offset-1 mt-5 p-2"> 
   <% if notice %>
-  <p class="alert alert-success"><%= notice %></p>
+    <p class="alert alert-success"><%= notice %></p>
   <% end %>
   <% if alert %>
-  <p class="alert alert-danger"><%= alert %></p>
+    <p class="alert alert-danger"><%= alert %></p>
   <% end %>
-    <div class="row row-cols-1 shadow p-3 mb-5 bg-body opacity-75"> 
-        
+    <div class="row row-cols-1 shadow p-3 mb-5 bg-body opacity-75">  
       <div class="border-bottom mb-3"> 
         <h5><img src="<%= @category.icon%>" ><span><%= @category.name %></span></h5>  
       </div> 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/232198800-0f0a9587-eff7-4f49-a731-db825d3f95b0.png)

- [ ] It would be nice if your app handles the icons of the categories :+1: because I have added an icon and your app is not properly handling it :fearful: Kindly fix this issue to make your project professional :+1: